helmfile/Dockerfile

```diff
@@ -15,10 +13,8 @@ WORKDIR /
 
 RUN helm plugin install https://github.com/mumoshu/helm-x --version v${HELM_X_VERSION}
 
-ADD https://storage.googleapis.com/kubernetes-release/release/v${EKS_VERSION}/bin/linux/amd64/kubectl /usr/local/bin/kubectl
-RUN chmod +x /usr/local/bin/kubectl
-
-RUN curl -o /usr/local/bin/aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/${EKS_VERSION}/${EKS_DATE}/bin/linux/amd64/aws-iam-authenticator && \
-    chmod +x /usr/local//bin/aws-iam-authenticator
```

Comment:
@cw-sakamoto 
I started thinking that both `awscli` and `aws-iam-authenticator` could be installed.
It is important that both work for compatibility.

Reply:
@cw-ozaki 
Certainly, depending on how `kubeconfig` is written, it will be decided which one to use (of course, it should be corrected to a uniform way of writing...), so it is good to have both.